.github/workflows/notebook_tests.yml

```diff
@@ -9,7 +9,8 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: [3.7]
-
+    env:
```

Comment:
What does this do?

Reply:
This just adds an environment flag for the notebook which disables the visualisation during testing